src/applications/vaos/services/mocks/index.js

```diff
@@ -271,6 +271,44 @@ const responses = {
       },
     });
   },
+  'GET /vaos/v2/eligibility': (req, res) => {
+    const isDirect = req.query.type === 'direct';
+    const ineligibilityReasons = [];
+
+    if (
+      isDirect &&
+      (!req.query.facility_id.startsWith('984') ||
+        req.query.clinical_service_id === 'primaryCare')
+    ) {
+      ineligibilityReasons.push({
+        coding: [
+          {
+            code: 'ineligible-history',
```

Comment:
I don't know what these codes are yet, so will need to update after that happens